src/assets/i18n/en.json5

```diff
@@ -5386,6 +5386,24 @@
 
   "admin.system-wide-alert.title": "System-wide Alerts",
 
+  "search.advanced.filters.head": "Advanced Search",
+  "filter.search.operator.placeholder": "Operator",
+  "filter.search.text.placeholder": "Search text",
+  "filter.search.placeholder": "Select filter",
+  "search.filters.operator.equals.text": "Equals",
+  "search.filters.operator.notequals.text": "Not Equals",
+  "search.filters.operator.notcontains.text": "Not Contains",
+  "search.filters.operator.contains.text": "Contains",
+  "search.filters.filter.title.text": "Title",
+  "search.filters.filter.author.text": "Author",
+  "search.filters.filter.subject.text": "Subject",
+  "discover.filters.head": "Discover",
+  "community.search.head": "Search Within",
+  "collection.page.browse.search.head": "Search Within",
+  "search.filters.applied.f.title": "Title",
+  "search.filters.filter.entityType.text": "Item Type",
```

Comment:
I *think* these two `search.filters` keys are also unused in this PR and can be removed.  But again, double check it please.